src/Restaurant.js

```diff
@@ -1,14 +1,15 @@
 import React from "react";
 
+import Orders from "./Order";
+
 const Restaurant = () => {
-  const pizzas = 0;
   return (
     <div>
       <h3>Restaurant Orders</h3>
       <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+        <Orders orderType={"Pizzas"} />
```

Comment:
👍 - this looks good :) How could you render each of the Orders without rendering the Order component 3 times in a row. Think of how you do this with other components.